apps/gustaf/create_stellarmatches_from_file.h

```diff
@@ -81,157 +93,170 @@ void _getShortIds(StringSet<TId> & shortIds, StringSet<TId> & longIds)
 
 // Takes the values from the localMatchStore and creates a Stellar match out of them.
 // The Stellar match is, sorted by read, appended to stQueryMatches. Reads are identified by their short query IDs
-// (sQueryIds), the correct database by their short database IDs (sDBIds), which are both given in the GFF file
+// (sQueryIds), the correct database by their short database IDs (databaseIds), which are both given in the GFF file
 template <typename TSequence, typename TId>
 bool _createStellarMatches(StringSet<TSequence> & queries,
                            StringSet<TId> const & sQueryIds,
                            StringSet<TSequence> & databases,
-                           StringSet<TId> const & sDBIds,
                            StringSet<TId> const & databaseIds,
                            LocalMatchStore<> & lmStore,
-                           StringSet<QueryMatches<StellarMatch<TSequence, TId> > > & stQueryMatches)
+                           StringSet<QueryMatches<StellarMatch<TSequence, TId> > > & stQueryMatches,
+                           unsigned & numThreads)
```

Comment:
I'll guess numThreads is rdonly? I suggest copy-by-value if you don't change this value within the function.